project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/vendor/aos/aos.cjs.js

```
@@ -0,0 +1,614 @@
+      node = el.node,
+      data = el.data;
+
+
+  var hide = function hide() {
+    if (!el.animated) return;
+
+    removeClasses(node, options.animatedClassNames);
+    fireEvent('aos:out', node);
+
+    if (el.options.id) {
+      fireEvent('aos:in:' + el.options.id, node);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)